src/ngraph_encapsulate_op.cc

```diff
@@ -69,9 +70,27 @@ class NGraphEncapsulateOp : public tf::OpKernel {
     // Create the backend
     if (m_ng_backend == nullptr) {
 #if defined(NGRAPH_EMBEDDED_IN_TENSORFLOW)
-      m_ng_backend = std::make_shared<ng::runtime::interpreter::INTBackend>();
+    	m_ng_backend = ng::runtime::Backend::create("INTERPRETER");
+#elif defined(NGRAPH_TF_BACKEND)
+        auto env_value = std::string(std::getenv("NGRAPH_TF_BACKEND")); 
```

Comment:
Check `nullptr` first before you can cast it to string
https://stackoverflow.com/questions/5867242/how-to-put-the-contents-of-getenv-into-a-string